docs/customization.md

```diff
@@ -95,7 +110,8 @@ The ignore file allows for better control over overwriting existing files than t
 Examples:
 
 ```sh
-# OpenAPI Generator Ignore
+# .openapi-generator-ignore
```

Comment:
This isn't a necessary change, as this file is not required to be named `.openapi-generator-ignore`.